render/topologies.go

```diff
@@ -116,6 +118,56 @@ var ContainerRenderer = MakeReduce(
 	),
 )
 
+type containerWithHostIPsRenderer struct {
+	Renderer
+}
+
+// Render produces a process graph where the ips for host network mode are set
+// to the host's IPs.
+func (r containerWithHostIPsRenderer) Render(rpt report.Report) RenderableNodes {
+	containers := r.Renderer.Render(rpt)
+	hosts := MakeMap(
+		MapHostIdentity,
+		SelectHost,
+	).Render(rpt)
+
+	for id, c := range containers {
+		networkMode, ok := c.Node.Latest.Lookup(docker.ContainerNetworkMode)
+		if !ok || networkMode != docker.NetworkModeHost {
+			continue
+		}
+
+		ips, ok := c.Node.Sets.Lookup(docker.ContainerIPs)
+		if ok && len(ips) > 0 {
```

Comment:
It makes it a bit confusing when reading (one may think there are actual cases in which this will happen). I would suggest removing it, but I don't have a very strong opinion.